### PR TITLE
protonvpn 4.6.0

### PR DESCRIPTION
**Pull request by jonhermansen**
This update is being prompted when I launch 4.5.0. There is a beta version as well, but I preferred to stay on the stable version that is provided as an update.

I also updated the livecheck strategy to skip beta versions

**Important:** *Do not tick a checkbox if you haven’t performed its action.* Honesty is indispensable for a smooth review process.

_In the following questions `<cask>` is the token of the cask you're submitting._

After making any changes to a cask, existing or new, verify:

- [x] The submission is for [a stable version](https://docs.brew.sh/Acceptable-Casks#stable-versions) or [documented exception](https://docs.brew.sh/Acceptable-Casks#but-there-is-no-stable-version).
- [x] `brew audit --cask --online <cask>` is error-free.
- [x] `brew style --fix <cask>` reports no offenses.

Additionally, **if adding a new cask**:

- [ ] Named the cask according to the [token reference](https://docs.brew.sh/Cask-Cookbook#token-reference).
- [ ] Checked the cask was not [already refused](https://github.com/search?q=repo%3AHomebrew%2Fhomebrew-cask+is%3Aclosed+is%3Aunmerged+&type=pullrequests) (add your cask's name to the end of the search field).
- [ ] `brew audit --cask --new <cask>` worked successfully.
- [ ] `HOMEBREW_NO_INSTALL_FROM_API=1 brew install --cask <cask>` worked successfully.
- [ ] `brew uninstall --cask <cask>` worked successfully.

---

### CODE REVIEW

File: Casks/p/protonvpn.rb [hunk 1]
```diff
@@ -1,15 +1,19 @@
...
   name "ProtonVPN"
   desc "VPN client focusing on security"
   homepage "https://protonvpn.com/"
 
+  # The Sparkle feed can contain items on the "beta" channel, so we restrict
+  # matching to the default channel.
   livecheck do
-    url "https://protonvpn.com/download/macos-update3.xml"
-    strategy :sparkle, &:short_version
+    url "https://protonvpn.com/download/macos-update4.xml"
```
Comment (khipp) [inline Casks/p/protonvpn.rb:13]:
```suggestion
    url "https://protonvpn.com/download/macos-update#{version.major}.xml"
```

Comment (jonhermansen) [reply Casks/p/protonvpn.rb:13]: Ah, I didn't even make the connection here to the major version now being included in the Sparkle URL path.

Previous 4.x releases were provided under update3.xml, so 🤷 I'm not sure what is correct here.

Comment (jonhermansen) [reply Casks/p/protonvpn.rb:13]: On second thought, it makes sense to include the major version in the Sparkle URL path. LGTM

**Comment by jonhermansen** [comment]
Thanks @khipp for the feedback. I've included this change in a subsequent commit with an explanation.

**Review by khipp — APPROVED** [review]
Thank you for your first contribution to `homebrew-cask`, @jonhermansen! 🎉

**Pull request merged**